### ISSUE TITLE
Could you tell me your best test accuracy on smallNORB dataset?

### ISSUE THREAD

**Issue by HJLYU0519** [maomran/CapsNet#1]
I have trid several different configurations based on this implemantation(https://github.com/yl-1993/Matrix-Capsules-EM-PyTorch).  
I could not get the similar performance mentioned in the original paper(about 2%). 
Thus, could you tell me your best test accuracy on smallNORB?  and is there some tips for training Matrix Capsule?

**Comment by maomran** [comment]
The best accuracy I got was 90%, I met the authors in ICLR 2018 and asked them how did they reach this test accuracy which they didn't write much about in the paper, they said that they used some manual hacks for step size which they didn't explain. They also mentioned that they will make the code available on github but they didn't.

**Comment by HJLYU0519** [comment]
Ｍany thanks for your helpful information. 
i would  send  an e-mail to ask this question to the authors. Ａnd  If i get any useful information, i would record that under this question.
In addition, what do you think of the matrix capsule network? I am preparing to do some research on the matrix-capsule  network,I really appreciate it, if you can give some advice or guidance, please.

**Comment by maomran** [comment]
It sounds promising since it tries to generalize the learning of features in the network, it is a good enhancement to CNNs since they are good at detecting low level features but they are still not good at connecting features together and form a proper high level interpretation, you have to basically overfit this part by introducing lots and lots of examples and lots of data augmentation which is not something that you do to teach a kid how to recognize cats from behind or side-viewed. Capsnet on the other hands, trys to learn this connection using routing which allows training with less parameters and examples. It still needs to be developed yet the concept is intuitive. 
To mention also, it is compute intensive and exploits a different dataflow of the traditional CNN structure.
